types/payment_voucher_test.go

```diff
@@ -0,0 +1,38 @@
+package types
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"github.com/filecoin-project/go-filecoin/address"
+)
+
+func TestEncodeThenDecode(t *testing.T) {
```

Comment:
Can you give this test a name that related it to the payment channel? The scope of this file is shared with the entire `types` package. Something like `TestPaymentVoucherEncodingRoundTrip`.